src/status_im/contexts/wallet/wallet_connect/modals/base_modal/view.cljs

```diff
@@ -0,0 +1,114 @@
+(ns status-im.contexts.wallet.wallet-connect.modals.base-modal.view
```

Comment:
Maybe not a modal, but a container component with child components sounds fine. As long as container and children and pure, we can mix-and-match as needed.

I also feel that the sign screen will not change much. There will steps before and after, but overall it will stay consistent.

Reply:
Yeah, a container sounds good. It could take care of the navigation header and the bottom authentication footer and the rest composed in individual screens I guess.

Reply:
Anyway, too late — I removed this component in the last commit :)